docs/host_binaries_solutions_comparison.md

```diff
@@ -0,0 +1,262 @@
+- supporting branches
+- easy way to manage files
+- easy to install and configure
+
+## git-lfs
+
+Docs can be found [here](https://git-lfs.github.com)
+
+### Costs
+
+[about-storage-and-bandwidth-usage](https://docs.github.com/en/github/managing-large-files/about-storage-and-bandwidth-usage)
+
+Each data pack cost is $5 per month. It contains:
+
+1. 50gb bandwidth
+2. 50gb storage
+
+### Is git lfs versioning files?
+
+Yes. Git LFS versioning files and show changed md5 sum and size.
+
+On main repository git diff looks like:
+
+```git
+
+diff --git a/app/build/outputs/apk/release/app-release-unsigned.apk b/app/build/outputs/apk/release/app-release-unsigned.apk
+index bba31d0..d165ca1 100644
+--- a/app/build/outputs/apk/release/app-release-unsigned.apk
++++ b/app/build/outputs/apk/release/app-release-unsigned.apk
+@@ -1,3 +1,3 @@
+ version https://git-lfs.github.com/spec/v1
+-oid sha256:bf48d577836f07a3d625ee30f04eb356c0b1158770f613df0d82b6ef40d300d3
+-size 1938709
++oid sha256:3349370934f8a1695b6ace6db53e58c0f24a1d9c02ce703b4a78870175c8c066
++size 1938769
+
+```
+
+### How to configure git lfs?
+
+To add a file extension to git lfs you should execute the following command:
+
+```bash
+
+git lfs track "*.apk"
+
+```
+
+Now all files with ``` .apk ``` extensions will be added to git lfs
+
+### How to add file
+
+The file should be added in `normal` way to git
+
+```git
+
+git add app-debug.apk
+git commit -m "Add apk file"
+git push origin master
+
+```
+
+### How to play with branches
+
+There is nothing to configure. When you modify files on branch changes not shown on the master.
+
+## git-submodules
+
+Docs can be found [here](https://gist.github.com/gitaarik/8735255)
+
+### Is git submodules versioning files?
+
+Git submodule is another git repository linked to the main repository. On main repository git diff looks like:
+
+```txt
+
+diff --git a/apks b/apks
+--- a/apks
++++ b/apks
+@@ -1 +1 @@
+-Subproject commit 7036bcb56be19490b4445a7e31e821e80b9ff870
++Subproject commit 7036bcb56be19490b4445a7e31e821e80b9ff870-dirty
+
+```
+
+So from the main repository, we don't see what files changed. To check details we need to execute git diff from the submodule.
+
+### Is git submodule versioning files?
+
+Yes. Git Submodule versioning binary files like standard git repository.
+
+### How to configure git submodule?
+
+1. Create a submodule repository
+
+2. On `main` execute repository git submodule add git@github.com:url_to/awesome_submodule.git path_to_awesome_submodule.
+
+3. Execute `git submodule init`
+
+### How to add file
+
+Goto submodule directory
+
+```git
+
+git add app-debug.apk
+git commit -m "Add apk file"
+git push origin master
+
+```
+
+### How to play with branches
+
+Sub-modules are not on a branch. They are just a pointer to a particular commit of the sub-modules repository.
```

Comment:
Actually there is an option to track specific branch on submodule.
https://stackoverflow.com/questions/9189575/git-submodule-tracking-latest

Reply:
Thanks, changed!